index.bs

```diff
@@ -3620,8 +3620,9 @@ A few abstract operations are used in this specification for utility purposes. W
 </div>
 
 <emu-alg>
+  1. Assert: _O_ is not *undefined*.
   1. Assert: ! IsPropertyKey(_P_) is *true*.
-  1. If _args_ was not passed, let _args_ be a new empty List.
+  1. Assert: _args_ is a List.
```

Comment:
If we did this following the "If _args_ was not passed, let _args_ be a new empty List." step then we could satisfy the expectation that args is optional.